kolibri/core/templates/kolibri/base.html

```diff
@@ -7,6 +7,7 @@
   {% base_frontend_async %}
   {% base_frontend_sync %}
   <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" />
+  <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" />
```

Comment:
I assume this is just for stubbing - CSS should be part of the webpack build pipeline.

If it's vital to get this in sooner rather than later, here is the documentation for it: https://webpack.github.io/docs/stylesheets.html - @DXCanas @indirectlylit and @aronasorman you may also be interested.